__tests__/app.test.js

```diff
@@ -161,3 +161,58 @@ describe("GET /api/reviews/:review_id/comments", () => {
     });
   });
 });
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("status:201 and returns newComment", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: "Cool game bro",
+    };
+    return request(app)
+      .post("/api/reviews/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { newComment } }) => {
+        expect(newComment.hasOwnProperty("review_id")).toBe(true);
+        expect(newComment.hasOwnProperty("votes")).toBe(true);
+        expect(newComment.hasOwnProperty("author")).toBe(true);
+        expect(newComment.hasOwnProperty("body")).toBe(true);
+        expect(newComment.hasOwnProperty("created_at")).toBe(true);
+      });
+  });
+});
+test("Status 400 Bad request, needs username and body", () => {
+  const newComment = { body: "Bad game bro" };
+  return request(app)
+    .post("/api/reviews/1/comments")
+    .send(newComment)
+    .expect(400)
+    .then(({ body: { msg } }) => {
+      expect(msg).toBe("Bad request");
+    });
+});
+test("Status 400 Bad request, needs valid username from users.js file.", () => {
```

Comment:
I appreciate it is a grey area but this would ideally be a 404 - user "not found".